lib/model/query/entities.js

```diff
@@ -491,13 +517,31 @@ const _processSubmissionEvent = (event, parentEvent) => async ({ Audits, Dataset
       // 2. baseVersion is empty and failed to parse in the update case
       // This second one is a special case related to issue c#727
       if ((err.problemCode === 404.8 || (err.problemCode === 400.2 && err.problemDetails.field === 'baseVersion')) && attemptCreate) {
-        maybeEntity = await Entities._createEntity(dataset, entityData, submissionId, submissionDef, submissionDefId, event, parentEvent, forceOutOfOrderProcessing);
+        maybeEntity = await Entities._createEntity(dataset, entityData, submissionId, submissionDef, submissionDefId, event, parentEvent, forceOutOfOrderProcessing, true);
       } else {
         throw (err);
       }
     }
-  else if (entityData.system.create === '1' || entityData.system.create === 'true')
-    maybeEntity = await Entities._createEntity(dataset, entityData, submissionId, submissionDef, submissionDefId, event, parentEvent, forceOutOfOrderProcessing);
+  else if (entityData.system.create === '1' || entityData.system.create === 'true') {
+    // note i dont think (???) forceOutOfOrderProcessing will ever be true here?
```

Comment:
That makes sense to me. Only entity updates are force-processed from the backlog, and entity updates are fully handled above. Explanatory comments might be helpful though.